provider/src/main/java/com/cloudera/director/google/sql/GoogleCloudSQLProviderConfigurationProperty.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (c) 2015 Google, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.cloudera.director.google.sql;
+
+import com.cloudera.director.spi.v1.model.ConfigurationProperty;
+import com.cloudera.director.spi.v1.model.ConfigurationPropertyToken;
+import com.cloudera.director.spi.v1.model.util.SimpleConfigurationPropertyBuilder;
+
+/**
+ * Google Cloud SQL configuration properties.
+ */
+public enum GoogleCloudSQLProviderConfigurationProperty implements ConfigurationPropertyToken {
+
+  // TODO(kl3n1nz) Might want to make an API call to list valid values.
+  REGION(new SimpleConfigurationPropertyBuilder()
+      .configKey("region_sql")
```

Comment:
Renaming the enum constant is fine, but it was actually the config key itself that seemed to be inconsistent with the naming scheme in the rest of the plugin. I agree that it needs to be different from the corresponding config key on the compute side. I don't feel strongly about this, though.